### PR TITLE
Add some apps

### PR DESCRIPTION
**Pull request by gdonisi**
Hi, I added some new apps to the list.

<details>
<summary>QR Scanner (Privacy Friendly)</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/com.secuso.privacyFriendlyCodeScanner/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/com.secuso.privacyFriendlyCodeScanner/).
- [ ] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Parlera — word guessing game</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/com.enjoyingfoss.parlera/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/com.enjoyingfoss.parlera/).
- [ ] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Sudoku (Privacy Friendly)</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/org.secuso.privacyfriendlysudoku/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/org.secuso.privacyfriendlysudoku/).
- [x] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Did I Take My Meds?</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/dev.corruptedark.diditakemymeds/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/dev.corruptedark.diditakemymeds).
- [x] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Feeel - home workouts</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/com.enjoyingfoss.feeel/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/com.enjoyingfoss.feeel/).
- [x] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Just Another Workout Timer</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/com.blockbasti.justanotherworkouttimer/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/com.blockbasti.justanotherworkouttimer/).
- [x] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Pedometer (Privacy Friendly)</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/org.secuso.privacyfriendlyactivitytracker/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/org.secuso.privacyfriendlyactivitytracker/).
- [ ] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>Catima</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/me.hackerchick.catima/latest/).
- [x] It has 0 Droid [Anti-Features](https://f-droid.org/packages/me.hackerchick.catima/).
- [x] Has dark theme
- [x] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

<details>
<summary>OsmAnd</summary>

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/net.osmand.plus/latest/).
- [ ] It has 0 Droid [Anti-Features](https://f-droid.org/packages/net.osmand.plus/).
- [x] Has dark theme
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

</details>

I don't know if "Health" is okay for both workout and medicine, I didn't create two categories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,6 +240,14 @@ __AWESOME__ apps counter: __121__ 🎉
 - [x] [GitHub](https://github.com/wewewe718/QrAndBarcodeScanner)
 - [ ] Official page
 
+### QR Scanner (Privacy Friendly)
```
Comment (Psyhackological) [inline README.md:243]: Unfortunately for now I cannot add this.

**Review by Psyhackological — CHANGES_REQUESTED** [review]

**Comment by gdonisi** [comment]
Feel free to remove apps you don't think are okay for the list.
You can configure OsmAnd dark theme in Configure profile - General Settings - App theme.  Roads are still light, unfortunately, but the rest of the app is dark.

File: README.md [hunk 2]
```diff
@@ -645,13 +671,46 @@ __AWESOME__ apps counter: __121__ 🎉
...
+## Health
+### Did I Take My Meds?
+<img alt="DidITakeMyMeds?" height="64" src="https://raw.githubusercontent.com/CorruptedArk/did-i-take-my-meds/master/fastlane/metadata/android/en-US/images/icon.png">
+
+- [x] [Google Play](https://play.google.com/store/apps/details?id=dev.corruptedark.diditakemymeds)
+- [x] [F-Droid](https://f-droid.org/packages/dev.corruptedark.diditakemymeds)
+- [x] [GitHub](https://github.com/CorruptedArk/did-i-take-my-meds)
+- [ ] Official page
+
+### Feeel - home workouts
```
Comment (Psyhackological) [inline README.md:690]: Everything is fine but I am thinking about `Workout` category in contents instead of `Health`.

File: README.md [hunk 3]
```diff
@@ -645,13 +671,46 @@ __AWESOME__ apps counter: __121__ 🎉
...
+<img alt="Feeel" height="64" src="https://gitlab.com/enjoyingfoss/feeel/-/raw/master/fastlane/metadata/android/en-US/images/icon.png">
+
+- [x] [Google Play](https://play.google.com/store/apps/details?id=com.enjoyingfoss.feeel)
+- [x] [F-Droid](https://f-droid.org/packages/com.enjoyingfoss.feeel/)
+- [x] [GitLab](https://gitlab.com/enjoyingfoss/feeel)
+- [ ] Official page
+
+### Just Another Workout Timer
```
Comment (Psyhackological) [inline README.md:698]: Same here as in `Feeel`.

File: README.md [hunk 4]
```diff
@@ -720,6 +779,15 @@ __AWESOME__ apps counter: __121__ 🎉
 - [x] [GitHub](https://github.com/florisboard/florisboard)
 - [ ] Official page
 
+## Loyalty Card
+### Catima ❤️
```
Comment (Psyhackological) [inline README.md:783]: Could you please explain me how does this app work from the practical standpoint?

File: README.md [hunk 5]
```diff
@@ -729,6 +797,14 @@ __AWESOME__ apps counter: __121__ 🎉
 - [x] [GitHub](https://github.com/organicmaps/organicmaps)
 - [x] [Official page](https://organicmaps.app/)
 
+### OsmAnd
```
Comment (Psyhackological) [inline README.md:800]: Could you help me find the dark theme mode in this app?

**Review by Psyhackological — CHANGES_REQUESTED** [review]

**Comment by Psyhackological** [comment]
Make these required changes and I'm going to merge this. Thanks!

**Pull request merged**